src/components/ComposedModal/_composed-modal.scss

```
@@ -0,0 +1,59 @@
+@import '../../globals/vars';
+@import '../../globals/layout';
+
+.#{$iot-prefix}--composed-modal--modal {
+  .bx--modal-container {
+    @media (min-height: carbon--rem(515px)) {
+      overflow-y: auto;
+    }
+  }
+
+  /* support large modals for ll the sizes */
+  &.big-modal {
+    .bx--modal-header {
+      margin-bottom: 0rem;
```

We can leave off the units when value is zero
```suggestion
      margin-bottom: 0;
```